node-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,165 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'node',
+        rules: [
+          {
+            alert: 'NodeFilesystemSpaceFillingUp',
+            expr: |||
+              predict_linear(node_filesystem_avail{%(nodeExporterSelector)s,%(fsSelectors)s}[6h], 24*60*60) < 0
+                AND
```

Comment:
operators should be lowercase.

Also the indentation convention is the other way around, operands are indented relative to the operator

Reply:
This is what we use throughout the examples on prometheus.io, and always have. https://prometheus.io/docs/practices/rules/ for example.